index.js

```diff
@@ -8,6 +8,7 @@ export default function parse (parse, from) {
 
   const text = String(parse).toLowerCase()
   const date = new Date((isFinite(from) && text.indexOf('now') === -1) ? Number(from) : Date.now())
+  date.setMilliseconds(0)
```

Comment:
super! 👍  Maybe a bit OCD can we move it to line 15, just before `Object.keys(match)` so all variables is declared on top? 🙈

Reply:
+1, please move it to line 15 😛